src/aspire/volume/__init__.py

```diff
@@ -461,40 +468,47 @@ def _gaussian_blob_Cn_vols(
     :return: A Volume instance containing C Gaussian blob volumes with Cn symmetry.
     """
 
-    def _eval_gaussian_blobs(L, Q, D, mu, order, dtype=np.float64):
-        try:
-            order = int(order)
-        except Exception:
-            raise NotImplementedError(
-                f"C{order} symmetry not supported. Only Cn symmetry, where n is an integer, is supported."
-            )
-
+    # Evaluate the symmetrized gaussian blobs.
+    def _eval_gaussians(L, K, Q, D, mu, dtype=np.float64):
         g = grid_3d(L, dtype=dtype)
         coords = np.array(
             [g["x"].flatten(), g["y"].flatten(), g["z"].flatten()], dtype=dtype
         )
 
-        K = Q.shape[0]
+        n_blobs = Q.shape[0]
         vol = np.zeros(shape=(1, coords.shape[-1])).astype(dtype)
-        rot = np.zeros(shape=(order, 3, 3)).astype(dtype)
 
-        angles = np.zeros(shape=(order, 3))
-        angles[:, 2] = 2 * np.pi * np.arange(order) / order
-        rot = Rotation.from_euler(angles).matrices
-
-        for k in range(K):
-            for j in range(order):
-                coords_k = rot[j, :, :] @ coords - mu[k, :, np.newaxis]
-                coords_k = (
-                    np.eye(3) / np.sqrt(np.diag(D[k, :, :])) @ Q[k, :, :].T @ coords_k
-                )
+        for k in range(n_blobs):
+            coords_k = coords - mu[k, :, np.newaxis]
+            coords_k = (
+                Q[k].T @ coords_k * np.sqrt(1 / np.diag(D[k % K, :, :]))[:, np.newaxis]
```

Comment:
During the symmetrize step we make rotated copies of Q: Q, R1Q, R2Q,...,RnQ where n is the cyclic order and each RiQ is a stack of length K.  The `% K` was just a way to divide each RiQ by the same stack of variances, D. 

I'm realizing now that since `% K` is just an accommodation of the symmetrize step it would probably be better to package `D` appropriately in `_symmetrize_gaussians`.